pysys/utils/stringutils.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# PySys System Test Framework, Copyright (C) 2006-2019 M.B. Grieve
```

Comment:
Opening a potential can of worms here but assigning copyright on a new file to someone other than its author is probably not what you intended here?  Can of worms because this might not be the only such instance and if we hope to get more people contributing to PySys some consideration of copyright assignment and licencing more generally might be needed.  As you know this is always a painful topic...

Reply:
Yep it's a tricky one... in this case I wrote the new lines and am happy at least for this to give Moray the copyright (given copyright owner!=author necessarily, and that the license is what preseves the relevant rights). Having a variety of copyright messages in different files could get messy to maintain. 

It'll be worth having a discussion about in future, especially as you say with hopefully more people contributin, but I'd rather having that discussion at leisure and perhaps face2face with interested parties (maybe in a pub with Moray :) ). But I'd like to punt this to after the current release.